### PR TITLE
test: re-add edit test as a community test

### LINKED ISSUE #9237: [Automation] implement edit message in community channel
### Description

The scenario `The user can edit a message` was removed because we no longer support public chats. 

Re-implement it in the context of a community channel.

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #9237

### CODE REVIEW

**Review by mprakhov — APPROVED** [review]
👍

**Comment by jrainville** [comment]
CI tests are actually green. Waiting for an approval to merge

**Review by noeliaSD — APPROVED** [review]
LGTM! Also tested locally!

**Pull request merged**
